src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -525,6 +527,27 @@ internal Collection<FunctionDescriptor> ReadFunctions(List<FunctionMetadata> met
             return functionDescriptors;
         }
 
+        private List<FunctionMetadata> ApplyMaximumFunctionCount(List<FunctionMetadata> metadatas)
```

Comment:
Yeah, "metadatas" was a relic from a large scale refactoring I did a long time ago. I didn't like it either :)